pkgs/tools/networking/nebula/default.nix

```
@@ -17,7 +17,7 @@ buildGoModule rec {
 
   subPackages = [ "cmd/nebula" "cmd/nebula-cert" ];
 
-  buildFlagsArray = [ "-ldflags='-X main.Build=${version}'" ];
+  buildFlagsArray = [ "-ldflags=" "-X main.Build=${version}" ];
```

This is exactly the same as

```suggestion
   buildFlagsArray = [ "-ldflags= -X main.Build=${version}" ];
```

which is why it works. The previous version passed literal single quotes to ldflags.

See https://github.com/NixOS/nixpkgs/issues/112054#issuecomment-774683550 why `buildFlagsArray` is misleading.

---

Should we make one string than?

---

I would move it to bash, until we have `__structuredAttrs`. That will work in all cases so it will be much less confusing than doing `buildFlagsArray` in Nix.

---

Sound like a major refactoring with unclear benefits to me, when we have `__structuredAttrs` we would need to move it back.